scaladoc/test-source-links/dotty/tools/scaladoc/source-links/RemoteLinksTest.scala

```diff
@@ -36,16 +37,23 @@ class RemoteLinksTest:
   @Test
   def runTest =
     assertTrue(mtslAll.nonEmpty)
-    val mtsl = randomGenerator.shuffle(mtslAll).take(20) // take 20 random entries
+    val mtsl = randomGenerator.shuffle(mtslAll).take(40) // take 40 random entries
     val pageToMtsl: Map[String, List[(String, String)]] = mtsl.groupMap(_._2.split("#L").head)(v => (v._1, v._2.split("#L").last))
     pageToMtsl.foreach { case (link, members) =>
       try
         val doc = getDocumentFromUrl(link)
-        members.foreach { (member, line) =>
+        println(s"Checking $link")
+        members.foreach { case (member, expectedLine) =>
           if !member.startsWith("given_") then // TODO: handle synthetic givens, for now we disable them from testing
-            val loc = doc.select(s"#LC$line").text
+            val toLine = expectedLine + 3
             val memberToMatch = member.replace("`", "")
-            assertTrue(s"Expected to find $memberToMatch at $link at line $line", loc.contains(memberToMatch))
+            val lineCorrectlyDefined = (expectedLine.toInt until toLine.toInt).exists{ line =>
```

Comment:
If I understand it correctly we look for the member in the range of lines. Was there something failing for the exact line match?

Reply:
yes, in was the case of name that spans in multiple lines like `asMatchable`:  https://github.com/lampepfl/dotty/blob/3.0.0/library/src/scala/compiletime/package.scala#L169-L170